addons/calendar/views/calendar_views.xml

```diff
@@ -313,6 +313,7 @@
                         <field name="stop_date" string="To" attrs="{'required': [('allday','=',True)],'invisible': [('allday','=',False)]}"/>
                         <field name="start" string="From" attrs="{'required': [('allday','=',False)], 'invisible': [('allday','=',True)]}" options="{'block': True, 'icon': 'fa fa-users'}"/>
                         <field name="stop" string="To" attrs="{'invisible': [('allday','=',True)]}"/>
+                        <field name="duration" string="duration" attrs="{'invisible': True}"/>
```

Comment:
What happens when you create a database in 16.3, and then checkout this branch without reloading the views? Does it crash in the javascript ?It is important because most people will update the code without reloading all the views. It must be bullet proof.